Wildfly integration tests are failing

##### Issue Overview

The wildfly integration tests are failing with:

```
Error:  Failed to execute goal org.wildfly.plugins:wildfly-maven-plugin:5.0.0.Final:provision (provision-wildfly) on project integration-tests: Execution provision-wildfly of goal org.wildfly.plugins:wildfly-maven-plugin:5.0.0.Final:provision failed: Invalid Manifest
Error:  caused by:
Error:   Unknown schema version "1.1.0"
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :integration-tests
Error: Process completed with exit code 1.
```

##### Steps To Reproduce
Look at the current https://github.com/arquillian/arquillian-core/actions and you will see a few failed "Arquillian Integration Tests"
runs

--- a/.github/workflows/integration-tests.yml
+++ b/.github/workflows/integration-tests.yml
@@ -27,7 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest ]
-        java: ['11', '17', '21']
+        java: ['17', '21']
 
     steps:
       - uses: actions/checkout@v4

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -172,8 +172,8 @@
         <skip.provision.server>${skipTests}</skip.provision.server>
 
         <!-- Versions -->
-        <version.org.wildfly.arquillian>5.1.0.Beta4</version.org.wildfly.arquillian>
-        <version.wildfly-maven-plugin>5.0.0.Final</version.wildfly-maven-plugin>
+        <version.org.wildfly.arquillian>5.1.0.Beta7</version.org.wildfly.arquillian>
+        <version.wildfly-maven-plugin>5.1.0.Beta2</version.wildfly-maven-plugin>
 
         <jboss.home>${project.build.directory}${file.separator}wildfly</jboss.home>
       </properties>

